docs/reference/analysis/tokenizers/simplepattern-tokenizer.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[analysis-simplepattern-tokenizer]]
+=== Simple Pattern Tokenizer
+
+experimental[]
+
+The `simplepattern` tokenizer uses a regular expression to capture matching
+text as terms. The set of regular expression features it supports is more
+limited than the <<analysis-pattern-tokenizer,`pattern`>> tokenizer, but the
+tokenization is generally faster.
+
+This tokenizer does not support splitting the input on a pattern match, unlike
+the <<analysis-pattern-tokenizer,`pattern`>> tokenizer. To split on pattern
+matches using the same restricted regular expression subset, see the
+<<analysis-simplepatternsplit-tokenizer,`simplepatternsplit`>> tokenizer.
+
+This tokenizer uses http://lucene.apache.org/core//6_5_1/core/org/apache/lucene/util/automaton/RegExp.html[Lucene regular expressions].
```

Comment:
Yeah, I wasn't sure what to do here, this just seemed least-bad since I saw it in other places. Using that version property is much better but it looks like that url doesn't exist yet for `lucene_version_path = 7_0_0` where we are right now. 

My initial impression is that having that link broken until Lucene's 7.0.0 release is probably better than having it possibly be out of date forever, especially if lucene just isn't hosting a version 7 javadoc right now (I'll do a little more looking).

So I'll use the version property unless we really want to avoid that broken link (my reading of the [docs project](https://github.com/elastic/docs#building-part-of-a-book) is it only complains about broken links within the book, not external urls)